### PR TITLE
Putting SDK team knowledge into docs

### PR DESCRIPTION
**Pull request by stephanie-anderson**
Adding explanation of `before_send` limitations

### CODE REVIEW

**Review by sl0thentr0py — APPROVED** [review]

File: src/platforms/common/configuration/options.mdx [hunk 1]
```diff
@@ -690,6 +690,8 @@ thread-safe APIs and only use Unity-specific APIs after you've checked that you'
 
 This function is called with an SDK-specific message or error event object, and can return a modified event object, or `null` to skip reporting the event. This can be used, for instance, for manual PII stripping before sending.
 
+Please, note when <PlatformIdentifier name="before-send" /> is being executed, all scope data has already been applied to the event. Further modification of the scope won't have any effect.
```
Comment (shanamatthews) [inline src/platforms/common/configuration/options.mdx:693]:
```suggestion
By the time <PlatformIdentifier name="before-send" /> is executed, all scope data has already been applied to the event. Further modification of the scope won't have any effect.
```

**Review by shanamatthews — APPROVED** [review]
just a lil wording tweak :)

**Comment by stephanie-anderson** [comment]
> just a lil wording tweak :)

Thank you @shanamatthews!! This is exactly what I was hoping for xD 🙏

**Review by bruno-garcia — APPROVED** [review]
:shipit:

**Pull request merged**